spec/unit/AndroidManifest.spec.js

```diff
@@ -40,7 +40,7 @@ describe('AndroidManifest', () => {
         android:supportsRtl="true" android:debuggable="true">
         <activity android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale"
             android:label="@string/activity_name" android:launchMode="${ACTIVITY_LAUNCH_MODE}"
-            android:name="${ACTIVITY_NAME}" android:theme="@android:style/Theme.DeviceDefault.NoActionBar"
+            android:name="${ACTIVITY_NAME}" android:theme="@android:style/Theme.AppCompat.NoActionBar"
```

Comment:
The `android:` before style/Theme was also removed

https://github.com/apache/cordova-android/pull/1052/files#diff-964af6e8b782845fe05d5c1c69a192dc4571018c804b1766a7dc9566c9ac6b1aR38